framework/helpers.js

```diff
@@ -33,4 +35,28 @@ export default class Helpers {
         const elementHandle = await page.$(selector);
         return await elementHandle.contentFrame();
     }
+
+    async enableConsoleLogs() {
```

Comment:
New test should be added to helpers.test.js that asserts console errors are logged.